src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -929,8 +929,12 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
             DiagnosticBag diagnostics,
             bool needDeclaration = false)
         {
+            if (fieldSymbol.IsTupleField)
```

Comment:
Just curious. It is not necessarily a bad idea. 
I'd be worried if tuple members require special handling in lambdas and such. If no, then it seems ok to lower them in Emit.